en/config/safety.md

```diff
@@ -1,8 +1,10 @@
 # Safety Configuration (Failsafes) 
 
-PX4 has configurable failsafe systems to protect and recover your vehicle if something goes wrong! These allow you to specify areas and conditions under which you can safely fly, and the [action](#failsafe_actions) that will be performed if a failsafe is triggered (for example, landing, holding position, or returning to a specified point).
+PX4 has a number of safety features to protect and recover your vehicle if something goes wrong:
 
+* *Failsafes* allow you to specify areas and conditions under which you can safely fly, and the [action](#failsafe_actions) that will be performed if a failsafe is triggered (for example, landing, holding position, or returning to a specified point).
 The most important failsafe settings are configured in the *QGroundControl* [Safety Setup](#qgc_safety_setup) page. Others must be configured via [parameters](#failsafe_other).
+* [Safety switches](#safety_switch) on the remote control can be used to immediately crash or return the vehicle in the event of a problem. 
```

Comment:
Instead of crash I'd write 'stop all motors'. Because there are situations when you are on the ground (or really close to it) and you trigger it.